test/rails/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM brandonmathis/taperole:latest
```

Comment:
Actually. This is a docker base box that I defined which is a base install of ubuntu with ruby 2.3.0 and ansible.

[It's hosted here on docker hub](https://hub.docker.com/r/brandonmathis/taperole/)